app/scripts/controllers/item.js

```diff
@@ -4,172 +4,211 @@
  * Email item Controller -- The UI for the email pane
  */
 
-app.controller('ItemCtrl', [
-  '$scope', '$rootScope', '$routeParams', '$location', 'Email', '$http', '$cookies',
-  function ($scope, $rootScope, $routeParams, $location, Email, $http, $cookies) {
-    const iframe = null
```

Comment:
Lots of auto-formatting in this file - this is what was removed and we no longer use this higher level variable in the closure.